content/kubermatic/master/operation/etcd/etcd-launcher/_index.en.md

```diff
@@ -0,0 +1,105 @@
+
+{{% notice warning %}}
+etcd-launcher is an experimental feature. It should not be enabled unless all users clusters are operating nominally. It's possible that a user cluster etcd ring could fail to enable the feature if the etcd ring was not in a stable condition during the update.
+{{% /notice %}}
+
+Since this is an experimental feature, it's disabled by default. There are two modes to enable etcd-launcher support:
+## Enable etcd-launcher for a specific user cluster
+In this mode, the feature is only enabled for a specific user cluster. This can be done by editing the object cluster and enabling the feature flag for etcd-launcher:
+
+```bash
+$ kubectl edit cluster 27k9rzjzs7
+```
+
+Edit the cluster spec to add the `etcdLauncher` feature flag, and set it to `true`:
+
+```yaml
+spec:
+  features:
+    etcdLauncher: true
+```
+
+Once the cluster object is updated. The etcd StatefulSet will start updating the etcd ring pods one by one. Replacing the old etcd run command with the etcd-launcher.
+
+#### Disabling etcd-launcher
+In this mode, to disable etcd-launcher, you can simply remove the feature flag from the cluster spec, or set it to `false`.
+
+## Enable etcd-launcher for all user clusters
+In this mode, the feature is enabled on the seed controller manager level. The cluster feature flag will be added to all user clusters.
+
+To enable etcd-launcher, the related feature should be enabled in the seed controller command. To do that, edit your KPP `values.yaml` file to add the feature gate:
+
+```yaml
+kubermatic:
+  controller:
+    featureGates: "EtcdLauncher=true"
+```
+Next, you need run the kubermatic-installer to update the controller deployment:
+
+```bash
+$ ./kubermatic-installer deploy --config kubermatic.yaml --helm-values values.yaml
+```
+This should update the seed controller manager deployment and enable etcd-launcher for all users clusters.
+
+{{% notice note %}}
+Once the seed controller manager is reloaded, all users clusters will get upgraded to use etcd-launcher. In seed clusters with a large number of user clusters, this might take some time depending on the applied `-max-parallel-reconcile` value (default is 10).
+{{% /notice %}}
+
+
+#### Disabling etcd-launcher
+In this mode, to disable etcd-launcher, you need to revert the changes to the `values.yaml` file, and apply again. Additionally, you will need to disable it in each user cluster flag.
+
+To disable etcd-launcher for a specific cluster you need to set the cluster level feature flag to `false`. This can be applied _before_ enabling the feature gate on the seed controller manager to prevent the cluster from being upgraded in the first place.
+
+
+# Etcd Launcher Features
+Enabling etcd-launcher enables clusters operators to perform several operational tasks that were not possible before. With the the v2.15.0 releases, etcd-launcher provides the following capabilities:
```

Comment:
"cluster operators" I think, the other is double plural maybe? Not sure. Sounds weird though. ^^